doc/io.rst

```diff
@@ -25,7 +25,7 @@ module:
 
 .. ipython:: python
 
-    import cPickle as pickle
+    import pickle as pickle
```

Comment:
Good call, let's just do `import pickle`. We're not building the docs on readthedocs with Python 3 yet.